internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json

```diff
@@ -1428,555 +1428,18 @@
         "Count_CriminalIncidents_WeaponLawViolations_IsHateCrime",
         "Count_CycloneEvent",
         "Count_CycloneEvent_TropicalStorm",
-        "Count_Death",
```

Comment:
@pradh Can you confirm this deletion?

Reply:
Good eye.  Yes, that's from cl/453976812